backend/packages/Upgrade/src/api/middlewares/CORSMiddleware.ts

```diff
@@ -0,0 +1,33 @@
+import * as express from 'express';
+import { ExpressMiddlewareInterface, Middleware } from 'routing-controllers';
+import cors from 'cors';
+import { env } from '../../env';
+
+const ourCors = {
+  origin: function (origin, callback) {
+    // Allow requests with no origin (non-browser requests)
+    if (origin === undefined) {
```

Comment:
`null` would error on line 14, I don't know if it's possible for that to happen but `if (origin)` covers all the nullish scenarios.